addons/pinniped/config-controller/README.md

```diff
@@ -19,3 +19,17 @@ To run static checks (e.g., `go fmt`, `go vet`, `golangci-lint`) and the tests:
 ```sh
 ./hack/check.sh
 ```
+
+To generate the default pinniped addon secret:
+
+```sh 
+# manually via ytt
+ytt -f ./hack/ytt/default-package-secret.yaml -f ./hack/ytt/values.yaml -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere
+
+# using the generate script
+./hack/generate-package-secret.sh -v tkr=v1.23.3---vmware.1-tkg.1 -v iaas=vsphere 
+
+# using the make target from the top-level Tanzu Framework makefile
+# the variables here are slightly different to match what is expected in the rest of the build environment
+make generate-package-secret TKR_VERSION=v1.23.3---vmware.1-tkg.1 IAAS=vsphere
```

Comment:
same q as before - `TKR_VERSION` and `IAAS` seem like they are stale. do we think that this `make` invocation here will get out of date? i could see someone changing the root Makefile but forgetting to update this README.

Reply:
updated.